emukit/quadrature/kernels/integration_measures.py

```diff
@@ -105,8 +131,27 @@ def get_box(self) -> List[Tuple[float, float]]:
         """
         return self.bounds
 
+    def get_samples(self, num_samples: int, context_manager: ContextManager=None) -> np.ndarray:
+        """
+        samples from the uniform distribution.
+        :param num_samples: number of samples
+        :param context_manager: The context manager Contains variables to fix. If a context is given, this method
+        samples from the conditional distribution.
```

Comment:
Is this comment actually true? It does not seem to line up with implementation below, at least to my eyes

Reply:
not sure I understand. I sample from the joint and then set the context. You can do this because the dimensions of the joint are independent. thus p(x, y) = p(x)p(y). But possibly I am not seeing sth..